### ISSUE TITLE
[Bug]: Screen reader is not narrating status message information which appear after giving input in the 'Country' search field in Windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#10409]
### Link to reproducible scenario

https://www.ag-grid.com/javascript-data-grid/filtering/

### Describe the bug

Steps to reproduce 

1. Open the URL: [JavaScript Grid: Column Filters | AG Grid] (https://www.ag-grid.com/javascript-data-grid/filtering/) in latest edge browser.
2. 'Column filters' page will appear.
3. Press tab key to move to the 'country' column in table.
4. Press tab key to move to the 'filter' control and activate it using enter key. drop-down will appear and focus will be on the search edit field.
5. Give some input in the search edit field.
6. Verify whether screen reader is narrating status message information which appear after giving input in the 'Country' search field or not.

Actual behaviour
On giving input in the 'Search' edit field, screen reader remains silent and doesn't announce any information about the results.
Screen reader remains silent.

https://github.com/user-attachments/assets/2b98112d-d0da-4300-9794-ec148626c733

<img width="959" alt="Image" src="https://github.com/user-attachments/assets/488be5a7-8a25-4361-a36c-097f7a14f094" />

Expected behaviour
On giving input in the 'Search' edit field (Valid and invalid data) screen reader should announce status information in Windows.
Ex: 
1. On giving invalid data, screen reader should announce as 'No matches found/ No results found' information.
2. On giving valid data (Ali), screen reader should announce as '2 matches found/ 2 results found' information.

More information
Title: Screen reader is not narrating status message information which appear after giving input in the 'Country' search field in Windows
UserImpact: Screen reader user will not be able to know that triggered action has performed successfully or not if status message appearing on screen is not announced.

### Version

30.1.0

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by ava-utting** [comment]
Hi @msftedad ,

Thank you for reporting this.
We agree this would be a valuable addition to the product and we have added a feature request for this.

We have added this requirement to our backlog and we are tracking it with the following reference and description:
AG-9646 

We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid.

Thanks again for bringing this up with us.

Kind Regards,
Ava
